### ISSUE TITLE
KDE5 test isn't enabled for nixpkgs-unstable

### ISSUE THREAD

**Issue by danbst** [NixOS/nixpkgs#17393]
## Issue description
I've tried to run KDE5. I've recently updated nixpkgs-unstable channel and got the following error:

```
building Nix...
building the system configuration...
error: attribute ‘phonon-backend-gstreamer’ missing, at /nix/store/m1dmjkbbqhq14jwk892pn0mn2q845hf7-nixos-16.09pre87618.2fa9bd5/nixos/nixpkgs/nixos/modules/services/x11/desktop-managers/kde5.nix:151:9
(use ‘--show-trace’ to show detailed location information)
```

## Technical details

* System: 16.09pre87618.2fa9bd5 (Flounder)

**Comment by FRidh** [comment]
The bug has been fixed in 3dea00d. I don't know whether a test exists that should catch this.

**Comment by ttuegel** [comment]
Fixed in e5a79b0.
